lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
+      source_ref: "master",
```

Comment:
There is no need to keep the source_ref around if we don't use it internally. The goal of ExDoc.build_config is build a set of options and build a struct that is relevant internally, if we don't use source_ref past building the config, there is no point in keeping it in the struct.